strings/base_events.h

```diff
@@ -375,8 +375,37 @@ WINRT_EXPORT namespace winrt
         using delegate_type = Delegate;
 
         event() = default;
-        event(event<Delegate> const&) = delete;
-        event<Delegate>& operator =(event<Delegate> const&) = delete;
+
+        event(event<Delegate> const& value)
```

Comment:
Can just say `event` instead of `event<Delegate>` thanks to injected class names.

Reply:
Yeah, was existing so I left as is because it could have been a style choice.